front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/SpaceTimeChart.tsx

```diff
@@ -81,17 +81,17 @@ export default function SpaceTimeChart(props: SpaceTimeChartProps) {
   const rndContainerRef = useRef<Rnd>(null);
 
   const {
-    allowancesSettings = ORSD_GRAPH_SAMPLE_DATA.allowancesSettings,
+    allowancesSettings = ORSD_GRAPH_SAMPLE_DATA.allowancesSettings as AllowancesSettings,
     dispatchUpdateDepartureArrivalTimes = noop,
     dispatchUpdateMustRedraw = noop,
     dispatchUpdateSelectedTrainId = noop,
     dispatchUpdateTimePositionValues = noop,
     dispatchUpdateChart = noop,
     initialHeightOfSpaceTimeChart = 400,
-    inputSelectedTrain = ORSD_GRAPH_SAMPLE_DATA.selectedTrain as unknown as Train,
+    inputSelectedTrain = ORSD_GRAPH_SAMPLE_DATA.simulation.present.trains[0] as Train,
```

Comment:
This cast is not needed anymore
```suggestion
    inputSelectedTrain = ORSD_GRAPH_SAMPLE_DATA.simulation.present.trains[0],
```

Reply:
done